src/plugins/embeddable/public/lib/panel/panel_header/panel_actions/add_panel/add_panel_flyout.tsx

```diff
@@ -153,7 +153,10 @@ export class AddPanelFlyout extends React.Component<Props, State> {
         <EuiFlyoutHeader hasBorder>
           <EuiTitle size="m">
             <h2>
-              <FormattedMessage id="embeddableApi.addPanel.Title" defaultMessage="Add panels" />
+              <FormattedMessage
+                id="embeddableApi.addPanel.Title"
+                defaultMessage="Add from library"
```

Comment:
@elastic/kibana-app-services This is the only change I made to the embeddable plugin which updates the title of the add panel flyout to align with the rename of the button in Dashboard to `Add from library`.